sunpy/net/helio/hec.py

```diff
@@ -152,7 +157,10 @@ def search(self, *args, **kwargs):
             table = str.encode(table)
         start_time = qrdict['Time'].start
         end_time = qrdict['Time'].end
-        max_records = qrdict.get('max_records', 10)
+        max_records = qrdict.get('max_records', 50)
+        if max_records == 50:
+            log.info("Using default `max_records` of 50, this means you could up with fewer results than the "
+                     "web query. If you want to change this, set `a.helio.MaxRecords`")
```

Comment:
How about instead show a warning if the number of records returned by the search is equal to `max_records`? (At least if the default `max_records` is used.)

Reply:
yeah I agree 👍

Reply:
Will do.